djangocms_link/cms_plugins.py

```diff
@@ -1,46 +1,51 @@
 # -*- coding: utf-8 -*-
-from __future__ import absolute_import, print_function, unicode_literals
```

Comment:
`unicode_literals` should always be at the top as a [best practice](https://docs.djangoproject.com/en/1.10/topics/python3/#unicode-literals)